src/bpffeature.cpp

```diff
@@ -391,6 +391,61 @@ bool BPFfeature::has_kprobe_multi()
   return *has_kprobe_multi_;
 }
 
+static int create_map(std::string name,
+                      enum bpf_map_type type,
+                      uint32_t key_size = sizeof(int),
+                      uint32_t value_size = sizeof(int),
+                      uint32_t max_entries = 1,
+                      uint32_t flags = 0)
+{
+#ifdef HAVE_BCC_CREATE_MAP
+  return bcc_create_map(
+      type, name.c_str(), key_size, value_size, max_entries, flags);
```

Comment:
`bcc_create_map` shouldn't be used anymore. Check `detect_map` to see how map creation should be implemented.